code/obj/item/gun/kinetic.dm

```diff
@@ -418,7 +418,7 @@ ABSTRACT_TYPE(/obj/item/gun/kinetic)
 	name = "Faith"
 	desc = "'Cause ya gotta have Faith."
 	icon_state = "faith"
-	force = 5.0
+	force = 6
```

Comment:
Maybe turn all of these into macros instead? i.e. `MELEE_DMG_PISTOL` or whatever. This makes it easier to tune them without having to go through all of the weapons again (and potentially missing something).